pkg/nsx/endpoint.go

```diff
@@ -297,7 +297,7 @@ func (ep *Endpoint) createAuthSession(certProvider auth.ClientCertProvider, toke
 	ep.client.Jar = jar
 	ep.Unlock()
 	ep.setStatus(UP)
-	log.Info("session creation succeeded", "endpoint", u.Host)
+	log.V(2).Info("session creation succeeded", "endpoint", u.Host)
```

Comment:
session created info is also useful for debug auth related. Few of them will be output. suggest to keep Info

Reply:
ok